Find a functionally equivalent code:android.print.PrintManager.getEnabledPrintServices:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Gets the list of enabled print services.
 *
 * @return The enabled service list or an empty list.
 * @hide
 */

@@ -1,4 +1,8 @@
 {
+    if (mService == null) {
+        Log.w(LOG_TAG, "Feature android.software.print not available");
+        return Collections.emptyList();
+    }
     try {
         List<PrintServiceInfo> enabledServices = mService.getEnabledPrintServices(mUserId);
         if (enabledServices != null) {

